src/pages/index.js

```diff
@@ -5,52 +5,57 @@ import UserInfo from "../Components/UserInfo.js";
 import PopupWithForm from "../Components/PopupWithForm.js";
 import '../styles/style.css';
 import PopupWithImage from "../Components/PopupWithImage.js";
+import PopupWithConfirm from "../Components/PopupWithConfirm.js";
+import { config, API } from "../utils/constants.js";
 
-const config = {
-    formSelector: ".modal__form",
-    inputSelector: ".modal__input",
-    submitButtonSelector: ".modal__button",
-    inactiveButtonClass: "modal__button_disabled",
-    inputErrorClass: "modal__input_type_error",
-    errorClass: "modal__error_visible"
-};
-
-const initialCards = [
-    {
-        name: "New York",
-        link: "https://plus.unsplash.com/premium_photo-1681803606255-cb66b02f2b56?q=80&w=2574&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
-    },
-    {
-        name: "Chicago",
-        link: "https://images.unsplash.com/photo-1494522855154-9297ac14b55f?q=80&w=2670&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
-    },
-    {
-        name: "Golden Gate",
-        link: "https://images.unsplash.com/photo-1574054394439-06aef14b154e?q=80&w=2574&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D",
-    },
-    {
-        name: "Bald Mountains",
-        link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-    },
-    {
-        name: "Latemar",
-        link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-    },
-    {
-        name: "Yosemite Valley",
-        link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-    }
-]
-
-const renderCard = (card) => {
-    return new Card(card, cardTemplate, openImageModal).createCard();
-}
+const confirmPopup = new PopupWithConfirm('.modal[id=confirmModal]');
+confirmPopup.setEventListeners();
 
 const cardTemplate = document.querySelector('#card');
-const container = new Section({items: initialCards, renderer: renderCard }, ".places" );
-container.renderItems();
-
+const renderCard = (cardData) => {
+    return new Card(cardData, cardTemplate, openImageModal, 
+    (data, deleteButton) => {
+        confirmPopup.onPopupConfirm(() => {
+            return API.deleteCard(data._id)
+            .then(() => {
+                deleteButton.closest('.places__card').remove();
+            })
+            .catch((rej) => {
+                console.log(rej);
+            });
+        })
+    }, 
+    (data, heartElement) => {
+        if (!data.isLiked) {
+            API.addLike(data._id)
+            .then(() => {
+                data.isLiked = true;
+                heartElement.classList.add('places__heart_enabled');
```

Comment:
`                heartElement.classList.add('places__heart_enabled');
`

Now only class `Card` knows every **card selector**, so you need to encapsulate the likes logics inside its methods and call them in `index.js`. It’s a violation of **Single responsibility** principle